saleor/graphql/order/mutations/orders.py

```diff
@@ -9,11 +10,10 @@
 from ....core.tracing import traced_atomic_transaction
 from ....order import OrderLineData, OrderStatus, events, models
 from ....order.actions import (
+    _capture_payments,
```

Comment:
Shouldn't remove the underscore from the function? I would assume that the functions with `_` will be used only in local file.